docs/can-guides/commitment/recipes/canvas-clock/1-setup.js

```diff
@@ -1,23 +1,20 @@
-const ClockControlsVM = can.DefineMap.extend("ClockControlsVM", {
-  time: {
-    value({ resolve }) {
-      const intervalID = setInterval(() => {
-        resolve( new Date() );
-      },1000);
-
-      resolve( new Date() );
-
-      return () => clearInterval(intervalID);
-    }
-  }
-});
-
```

Comment:
New lines like this are significant in our guides because of the line highlighting we use to show what was added.